subprojects/vinumc/v_lib.h

```diff
@@ -1,8 +1,20 @@
 #ifndef __V_LIB_H__
 #define __V_LIB_H__
 
+#include <stdbool.h>
+#include <vutils/vec.h>
+
+struct v_str_vec VUT_VEC_DEF(char *);
+struct arg_vec VUT_VEC_DEF(int);
+struct name_vec VUT_VEC_DEF(char *);
```

Comment:
I don't like using vutils in the header. This adds an obligatory dependency to the developer of the external lib.

To help things, you could create a vec macro that gives you the base pointer and length to put it in the ctx.

Something like that:
```c
int* ptr;
size_t len;
VUT_VEC_GET_BASE_ARRAY(&vec, ptr, len)
```